platform/linux-generic/pktio/loop.c

```diff
@@ -195,10 +195,9 @@ static int loopback_recv(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 			/* Make sure there is enough data for the packet
 			 * parser in the case of a segmented packet. */
 			if (odp_unlikely(seg_len < PARSE_BYTES &&
-					 pkt_len > PARSE_BYTES)) {
-				odp_packet_copy_to_mem(pkt, 0, PARSE_BYTES,
-						       buf);
-				seg_len = PARSE_BYTES;
+					 pkt_len > seg_len)) {
+				seg_len = MIN(pkt_len, PARSE_BYTES);
```

Comment:
Include missing for `odp_macros_internal.h`.

Reply:
v4.